bitnami/postgresql/values.yaml

```diff
@@ -236,10 +236,10 @@ ldap:
   ## Ref: https://www.postgresql.org/docs/current/auth-ldap.html
   ##
   uri: ""
-## @param postgresqlDataDir PostgreSQL data dir folder
+## @param primaryDataDir PostgreSQL data dir folder
```

Comment:
Not sure if this change is expected. The value wasn't changed and Readme generator is failing